src/nadzoring/security/check_website_ssl_cert.py

```
@@ -724,11 +633,47 @@ def check_ssl_certificate(
+                "status": "error",
+                "error": "No certificate returned",
+                "verification": "failed" if verify else "unverified",
+            })
+        else:
+            result.update({
+                "remaining_days": None,
+                "status": "error",
+                "error": "Resolver error",
+                "verification": "failed" if verify else "unverified",
+            })
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)